src/event-hub.ts

```diff
@@ -37,15 +39,25 @@ export interface EventHubOptions extends LoggerOptions {
  */
 export class EventHub extends Logger {
   config: any;
+
```

Comment:
are those empty lines required by linting now? Looks kind of noisy to me...

Reply:
jep required by airbnb

https://eslint.org/docs/rules/lines-between-class-members

Reply:
Looks really ugly, I propose to add the exception for single line class members. What do the others say? @evannetwork/evan-developer 

> "exceptAfterSingleLine": false(default) do not skip checking empty lines after single-line class members

https://eslint.org/docs/rules/lines-between-class-members#rule-details

Reply:
i'm fine with that lets wait for feedback from the others.